app/src/utils/url_routes.js

```diff
@@ -1,4 +1,4 @@
-export const BASE = process.env.REACT_APP_URL_API + 'api/v1/';
+export const BASE = 'http://localhost:8000/api/v1/'; 
```

Comment:
Change it back to the env variable, with it we can set the API endpoint on production environments

Reply:
@gabibguedes are you using the dev environment or the docker?
Until the docker problem is solved, using the docker is not possible cause it doesnt reflect changes we made to the code. 
Also, as the docker doesnt work properly, we need this variable set this way as described on #157